docker-compose.yml

```diff
@@ -20,6 +20,7 @@ keycloak:
   volumes:
   - ./.docker-compose/keycloak.env:/etc/secrets/keycloak.env
   - ./govuk/:/opt/jboss/keycloak/themes/govuk/
+  - ./govuk-social-provider-login/:/opt/jboss/keycloak/themes/govuk-social-provider-login/
```

Comment:
Yes I think you are right. The second one also has the benefit of being slightly shorter.

Naming things is hard!